charts/catalog/templates/webhook-register.yaml

```diff
@@ -22,6 +22,58 @@ webhooks:
     apiGroups: ["servicecatalog.k8s.io"]
     apiVersions: ["v1beta1"]
     resources: ["clusterservicebrokers"]
+- name: mutating.clusterserviceclasses.servicecatalog.k8s.io
+  clientConfig:
+    caBundle: {{ b64enc $ca.Cert }}
+    service:
+      name: {{ template "fullname" . }}-webhook
+      namespace: "{{ .Release.Namespace }}"
+      path: "/mutating-clusterserviceclasses"
+  failurePolicy: Fail
+  rules:
+  - operations: [ "CREATE", "UPDATE" ]
```

Comment:
as discussed. We need to react on all mutating operation because we need to ensure that label will be synced. What's more, u need to change the internal behavior of webhook because right now webhook will return an error when it will be called for method different than CREATE or UPDATE.

Reply:
UPDATE method is handling PATCH request as well. So we don't have to modify anything